docs/using-draft-js.stories.mdx

```diff
@@ -0,0 +1,121 @@
+<Meta
+  title="Documentation/Using DraftJS"
+/>
+
+# Using Draft.js EditorState and ContentState
+
+## Contents
+[Introduction to Draft.js](#installation)
+
+* [EditorState](#editorstate)
+* [ContentState](#contentstate)
+* [Useful Links](#useful-links)
+
+## Installation
+The [TextEditor](https://carbon.sage.com/?path=/docs/design-system-text-editor--basic) and 
+[Note](https://carbon.sage.com/?path=/docs/design-system-note--basic) components utilise the `Draft.js` framework to 
```

Comment:
This `Note` link redirects to 

> https://carbon.sage.com/?path=/docs/alert--default

Reply:
That's because the note is in master yet

Reply:
I've put `DO NOT MERGE` on this PR for now, I was expecting `Note` to be in master by now